pkg/controller/apimanager/apimanager_controller.go

```diff
@@ -104,7 +104,7 @@ var _ reconcile.Reconciler = &ReconcileAPIManager{}
 
 // ReconcileAPIManager reconciles a APIManager object
 type ReconcileAPIManager struct {
-	operator.BaseControllerReconciler
+	*reconcilers.BaseReconciler
```

Comment:
Is there some motivation on passing a pointer instead of a value?

Reply:
Avoid unnecessary copy of the structures.